custom_components/tuya_local/devices/water_dispenser_kg1.yaml

```diff
@@ -0,0 +1,199 @@
+name: Inline Water Dispenser
+products:
+  - id: szx3eawxigxumxi5
+    name: KG1
+    manufacturer: Casdon
+    model: Y-GX1200-KG1
+
+entities:
+  - entity: switch
+    name: 出水开关
+    icon: mdi:water
+    dps:
+      - id: 8
+        type: boolean
+        name: switch
+
+  - entity: sensor
+    translation_key: status
+    class: enum
+    icon: mdi:cup-water
+    dps:
+      - id: 5
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: standby
+            value: 待机
+          - dps_val: refrigerating
+            value: 制冷中
+          - dps_val: drain_water
+            value: 出热水
+          - dps_val: drain_normal_water
+            value: 出常温水
+          - dps_val: drain_boiling_water
+            value: 出沸水
+          - dps_val: job_done
+            value: 出水完成
+          - dps_val: coldwater_prohibited
+            value: 禁止制冷
+          - dps_val: reset
+            value: 重置中
+          - dps_val: cleaning
+            value: 清洗中
+          - dps_val: cleaned
+            value: 清洗等待排水
+          - dps_val: empty_device
+            value: 排空中
+          - dps_val: emptied
+            value: 排空完成
+          - dps_val: drain_cool_water
+            value: 出冷水
+          - dps_val: high_temp_emptying
+            value: 高温排水中
+          - dps_val: high_temp_emptyed
+            value: 高温排水完成
+          - dps_val: water_shortage
+            value: 缺水
+
+  - entity: lock
+    name: 童锁
+    icon: mdi:lock
+    dps:
+      - id: 9
+        type: boolean
+        name: lock
+
+  - entity: switch
+    name: 自动冲洗
+    icon: mdi:autorenew
+    dps:
+      - id: 11
+        type: boolean
+        name: switch
+
+  - entity: switch
+    name: 制冷开关
+    icon: mdi:snowflake
+    dps:
+      - id: 102
+        type: boolean
+        name: switch
+
+  - entity: switch
+    name: 照明开关
+    icon: mdi:lightbulb
```

Comment:
icon should not be needed if you use a light entity. Name should also not be needed.